modules/blazingsql/src/execution_graph.ts

```diff
@@ -16,21 +16,22 @@ import {DataFrame, Series} from '@rapidsai/cudf';
 import {ExecutionGraphWrapper} from './node_blazingsql';
 
 export class ExecutionGraph {
-  private executionGraphWrapper: ExecutionGraphWrapper;
+  private executionGraphWrapper: ExecutionGraphWrapper|undefined;
 
-  constructor(executionGraphWrapper: ExecutionGraphWrapper) {
+  constructor(executionGraphWrapper?: ExecutionGraphWrapper) {
```

Comment:
We need some notion of an empty `ExecutionGraph` since we now return early here
https://github.com/rapidsai/node/blob/a54e601eb07eeeecf27c7635e7d3d6b0401af9c4/modules/blazingsql/src/blazingcontext.ts#L221

Any thoughts on representing an empty `ExecutionGraph`?

Reply:
We could have an `EmptyExecutionGraph` class?

Reply:
yeah simple enough, thanks.